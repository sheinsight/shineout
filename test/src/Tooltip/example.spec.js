import exampleTest from '../../example'

describe('Tooltip[snapshot]', () => {
  exampleTest('Tooltip')
})
