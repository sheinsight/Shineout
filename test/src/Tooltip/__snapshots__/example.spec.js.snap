// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Tooltip[site/pages/components/Tooltip/example-1-base.js] 1`] = `
<div>
  <noscript />
  <i />
  <noscript />
  <i />
  <noscript />
  <i />
  <noscript />
  <i />
</div>
`;

exports[`Snapshot test: Tooltip[site/pages/components/Tooltip/example-2-click.js] 1`] = `
<div>
  <noscript />
  <i />
  <noscript />
  <i />
  <noscript />
  <i />
  <noscript />
  <i />
</div>
`;

exports[`Snapshot test: Tooltip[site/pages/components/Tooltip/example-3-disabled-inner.js] 1`] = `
<div>
  <noscript />
  <span>
    <button>
      被禁用
    </button>
  </span>
</div>
`;
